scenaroone.js

```diff
@@ -11,7 +11,6 @@ module.exports = {
         this.assert.urlContains(response.value, '//*[@id="middleContainer"]/ul[1]/li[3]')
```

Comment:
what you are doing here is to check that the url has changed after clicking on random, which is good.
However in the `urlContains()`, the second parameter you should have added should be a message like `the url has changed after clicking on random`, and in the callback instead of verifying that the random button is still showing you can instead check that the `img` element is present